contrib/Docker/README.md

```diff
@@ -1,54 +1,62 @@
 # Docker
-Circle CI linux pch job uses the Dockerfile contained in the same folder as this README to create an image with the binaries built for linux and store that in the job artifacts.
-
-The instructions below expect basic knowledge about how to configure TrinityCore and how to use Docker:
-1. Click on the green ✔ next to each commit
-1. Scroll to "ci/circleci: pch" and click on "Details"
-1. Login to Circle CI if necessary. You might have to repeat again the previous steps after logging in to reach the correct page
-1. Click on "Artifacts" tab in Circle CI website
-1. Download the docker.tar.gz archive containing the docker image
-1. Load the image into your Docker host using
+
+The Circle CI Linux PCH job uses the Dockerfile contained in the same folder as this README to create an image with the binaries built for Linux, and stores that in the job artifacts.
+
+The instructions below expect a basic knowledge of how to configure TrinityCore and how to use Docker:
+1. Click the green tick ✔ next to each commit.
+1. Scroll to "ci/circleci: pch" and click "Details".
+1. Log in to Circle CI if necessary. You may have to repeat the previous steps after logging in, to reach the correct page.
+1. Click the "Artifacts" tab in the Circle CI website.
+1. Download the docker.tar.gz archive containing the docker image.
+1. Load the image into your Docker host, using the following command:
     ```
     docker load -i docker.tar.gz
     ```
-1. Copy the .conf files from TrinityCore GitHub repository to a local folder that will be passed as mapped volume to docker
-1. Set the MySQL host in the .conf files to use the unix socket of MySQL, i.e.: ``".;/var/run/mysqld/mysqld.sock;username;password;database``
-1. Set the "DataDir" config in worldserver.conf to ``"/trinity/data"``
-1. Start authserver or worldserver as desired, mapping the required volumes
-    - authserver
-        ```
-        docker run --entrypoint=authserver -it --volume=/host/path/to/configs:/home/circleci/project/bin/check_install/etc --volume=/var/run/mysqld/mysqld.sock:/var/run/mysqld/mysqld.sock -p=3724:3724 "image name"
-        ```
-    - worldserver
-        ```
-        docker run --entrypoint=worldserver -it --volume=/host/path/to/configs:/home/circleci/project/bin/check_install/etc --volume=/var/run/mysqld/mysqld.sock:/var/run/mysqld/mysqld.sock --volume=/path/to/data/directory:/trinity/data -p=8085:8085 "image name"
-        ```
-    Change the ports and other parameters as needed. Consult Docker documentation for additional details about possible configuration settings.
+
+1. Copy the .conf files from the TrinityCore GitHub repository to a local folder which will be passed on as a mapped volume to docker.
+1. Set the MySQL host in the .conf files to use the UNIX socket of MySQL, i.e.: `.;/var/run/mysqld/mysqld.sock;username;password;database`
```

Comment:
Upon reflection, should the command line be encapsulated in double quotes to match the .conf file content format?

Reply:
whatever looks ok on github is fine

Reply:
OK, well, I just wondered if that string is supposed to match the exact spelling/format in the .conf file. If you don't think it needs to be a carbon copy of that content, then I suppose it is fine as is.

Reply:
could you post here an example as comment of what you mean, maybe with the 2 examples so I can see the difference ?

Reply:
https://github.com/TrinityCore/TrinityCore/blob/9af6bf15aa2fe836c3ebba306eaaa8971f00fac4/src/server/worldserver/worldserver.conf.dist#L95-L111

Reply:
```suggestion
1. Set the MySQL host in the .conf files to use the UNIX socket of MySQL, i.e.: `".;/var/run/mysqld/mysqld.sock;username;password;database"`
```